src/ripple/core/ConfigSections.h

```diff
@@ -30,48 +30,48 @@ struct ConfigSection
     static std::string importNodeDatabase () { return "import_db"; }
 };
 
-// VFALCO TODO Rename and replace these macros with variables.
-#define SECTION_ACCOUNT_PROBE_MAX       "account_probe_max"
-#define SECTION_CLUSTER_NODES           "cluster_nodes"
-#define SECTION_DATABASE_PATH           "database_path"
-#define SECTION_DEBUG_LOGFILE           "debug_logfile"
-#define SECTION_ELB_SUPPORT             "elb_support"
-#define SECTION_FEE_DEFAULT             "fee_default"
-#define SECTION_FEE_OFFER               "fee_offer"
-#define SECTION_FEE_OPERATION           "fee_operation"
-#define SECTION_FEE_ACCOUNT_RESERVE     "fee_account_reserve"
-#define SECTION_FEE_OWNER_RESERVE       "fee_owner_reserve"
-#define SECTION_FETCH_DEPTH             "fetch_depth"
-#define SECTION_LEDGER_HISTORY          "ledger_history"
-#define SECTION_INSIGHT                 "insight"
-#define SECTION_IPS                     "ips"
-#define SECTION_IPS_FIXED               "ips_fixed"
-#define SECTION_NETWORK_QUORUM          "network_quorum"
-#define SECTION_NODE_SEED               "node_seed"
-#define SECTION_NODE_SIZE               "node_size"
-#define SECTION_PATH_SEARCH_OLD         "path_search_old"
-#define SECTION_PATH_SEARCH             "path_search"
-#define SECTION_PATH_SEARCH_FAST        "path_search_fast"
-#define SECTION_PATH_SEARCH_MAX         "path_search_max"
-#define SECTION_PEER_PRIVATE            "peer_private"
-#define SECTION_PEERS_MAX               "peers_max"
-#define SECTION_RPC_ADMIN_ALLOW         "rpc_admin_allow"
-#define SECTION_RPC_STARTUP             "rpc_startup"
-#define SECTION_SMS_FROM                "sms_from"
-#define SECTION_SMS_KEY                 "sms_key"
-#define SECTION_SMS_SECRET              "sms_secret"
-#define SECTION_SMS_TO                  "sms_to"
-#define SECTION_SMS_URL                 "sms_url"
-#define SECTION_SNTP                    "sntp_servers"
-#define SECTION_SSL_VERIFY              "ssl_verify"
-#define SECTION_SSL_VERIFY_FILE         "ssl_verify_file"
-#define SECTION_SSL_VERIFY_DIR          "ssl_verify_dir"
-#define SECTION_VALIDATORS_FILE         "validators_file"
-#define SECTION_VALIDATION_QUORUM       "validation_quorum"
-#define SECTION_VALIDATION_SEED         "validation_seed"
-#define SECTION_WEBSOCKET_PING_FREQ     "websocket_ping_frequency"
-#define SECTION_VALIDATORS              "validators"
-#define SECTION_VALIDATORS_SITE         "validators_site"
+static std::string const SECTION_ACCOUNT_PROBE_MAX       ("account_probe_max");
```

Comment:
Hey, Scott, here's a suggestion that may significantly expand the scope of your change.  All caps is completely appropriate for #defines.  Not so great for everything else.  So I think the names of these static const strings should be lower case.  That means you'll need to fix up all their references, of course.

My initial take, since I'm not familiar with this code, is you could make a `Section` namespace inside the `ripple` namespace and drop all of the names in there.  Then they'd be referenced like this: `Section::databasePath`.  Vinnie or David may suggest a better approach.  Or there may be a pre-existing class or namespace they should be inside.